### PR TITLE
React home

### PR DESCRIPTION
**Pull request by wabowen89**

### CODE REVIEW

**Review by SunkissedQueen — CHANGES_REQUESTED** [review]
Curious what is the difference between the current PascalCase react styling components and the components I see with the `.` separating each word. Ex: `<Nav.Item></Nav.Item>` and `<NavItem></NavItem>`?
Please look over the files for uniform indentation and proper nesting.

**Comment by kyloh1996** [comment]
Curious what is the difference between the current PascalCase react styling components and the components I see with the . separating each word. Ex: <Nav.Item></Nav.Item> and <NavItem></NavItem>?
There really isnt much of a difference. Nav.Item is from react suite and NavItem is from reactstrap. From What I gathered rsuite is just a newer version.

File: src/styles/NotFound.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+.pic1{
+height: 25vw;
+
+
+}
+.pic2{
+width: 30vw;
+margin-bottom: 43%;
+z-index: 1;
+margin-left: 23vw;
```
Comment (SunkissedQueen) [inline src/styles/NotFound.css:32]: look over indentation

File: src/styles/Footer.css [hunk 2]
```diff
@@ -13,11 +13,11 @@
   font-family: 'Alegreya';
 }
 
-.R1 {}
+/* .R1 {}
```
Comment (SunkissedQueen) [inline src/styles/Footer.css:16]: do you need this commented out

File: src/styles/New.css [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#form{
+    background-color: #eebd43;
+    padding-top: 9vw;
+   padding-bottom: 9vw;
```
Comment (SunkissedQueen) [inline src/styles/New.css:4]: look at indentation

**Comment by kyloh1996** [comment]
Indentation has been updated and pushed

File: src/styles/New.css (hunk 3 above)
Comment (SunkissedQueen) [inline src/styles/New.css:4]: Looks like you missed one

**Review by SunkissedQueen — CHANGES_REQUESTED** [review]
Look like you skipped one

**Review by SunkissedQueen — APPROVED** [review]

**Pull request merged**